File: real_time_crawler/Python/SingleQuery.py
import requests
from pprint import pprint

# Structure payload.
payload = {
    'source': 'universal',
    'url': 'http://ip.oxylabs.io',
}

# Get response.
response = requests.request(
    'POST',
    'https://data.oxylabs.io/v1/queries',
    auth=('user', 'pass1'),
    json=payload,
)

# Print prettified response to stdout.
pprint(response.json())

File: real-time-crawler/Nodejs/SingleQuery.js
const axios = require('axios');

const payload = {
  "source": "universal",
  "url": "http://ip.oxylabs.io",
}

axios.post('https://data.oxylabs.io/v1/queries', payload, {
  auth: {
    username: 'user',
    password: 'pass1'
  },
  ContentType: "application/json"
})
.then(({ data }) => console.log(data))
.catch(err => console.log(err))